source/trendfiguur/04_conclusies.Rmd

```diff
@@ -0,0 +1,38 @@
+# Conclusies
+
+Deelnemers aan deze studie gaven hun voorkeur aan voor trendfiguren.
+Daarnaast lieten we hen twee reeksen van drie trendfiguren interpreteren.
+De eerste reeks was de meest eenvoudige figuur met de schaal van de y-as als een index.
+De tweede reeks was in de voorkeursstijl van de deelnemer.
+
+**Globale voorkeur**
+
+De deelnemers hadden een lichte voorkeur voor een y-as met een index in plaats van een procentuele wijziging.
+Het merendeel vond de referentielijnen een meerwaarde.
+De annotatie van deze lijnen was eveneens belangrijk.
+Driekwart van de deelnemers koos voor de weergave van de onzekerheid, bij voorkeur met een eenvoudig band in plaats van een gradiënt.
+De indeling van de trends met behulp van symbolen werd door een derde van de deelnemers gesmaakt.
+
+Deelnemers die aangaven minder vertrouwd te zijn met cijfers en statistiek kiezen iets vaker voor een eenvoudigere figuur zonder referentielijnen en onzekerheid en met een index in plaats van procentuele wijziging op de y-as.
+Qua voorkeur voor symbolen was er geen verschil tussen de types deelnemers.
+
+**Effect op correcte inschatting**
+
+Beide reeksen figuren die we de deelnemers voorschotelden, hadden dezelfde onderliggende gegevens.
+Ze verschilden enkel in de manier van weergave.
+Hierdoor kunnen we een eerste inschatting maken van het effect van de weergave op de mogelijkheid om de figuur juist te interpreteren.
+Aangezien meer dan 90% van de deelnemers zowel een betrouwbaarheidsinterval wensten als bijkomende referentielijnen, kunnen we deze effecten niet van elkaar onderscheiden.
+We hebben ze daarom samen genomen als het effect van de tweede reeks.
+
+Bij de eerste reeks met de eenvoudige figuur gaven slechts vier van de tien deelnemers een correcte interpretatie van de figuur.
```

Comment:
Niet akkoord dat de interpretatie niet correct is als er te weinig info aanwezig is om de grafiek correct te interpreteren.  In zo'n geval is 'onduidelijk' of 'weet ik niet' volgens mij minstens even correct (of misschien zelfs correcter?) dan gokken wat een belangrijke wijziging zou zijn, of hoe groot het betrouwbaarheidsinterval zou zijn...